ceph_iscsi_config/lun.py

```diff
@@ -158,6 +158,72 @@ def rbd_size(self):
                         else:
                             self.changed = True
 
+    def rbd_setqos(self, qos):
+        """
+        set qos parameters
+        :param qos: qos parameters (dict) qos parameters to control the iops/bps rate
+        """
+        with rados.Rados(conffile=settings.config.cephconf,
+                         name=settings.config.cluster_client_name) as cluster:
+            with cluster.open_ioctx(self.pool) as ioctx:
+                with rbd.Image(ioctx, self.image) as rbd_image:
+                    lock_info = rbd_image.list_lockers()
+                    if lock_info:
+                        lockers = lock_info.get("lockers")
+                        for holder in lockers:
+                            try:
+                                rbd_image.break_lock(holder[0], holder[1])
```

Comment:
Does break_lock also blacklist the current lock holder or is that lock_break()?

If it does not blacklist the holder then we are going to end up in a situation where a client thinks it has the lock and is doing IO but is never properly fenced off.

The break_lock use case in lun.py is only when a lun is first added and it only breaks it's own locks and not other nodes, so it does not worry about blacklisting.